packages/shared/src/utils/validation/form-validations.ts

```diff
@@ -107,3 +108,6 @@ export const getValidationFunction = (rule: string) => {
     return (value: string, options: TOptions, values: Record<string, string | boolean>) =>
         !!func(value, options, values);
 };
+
+export const shouldHideOccupationField = (employment_status?: string) =>
+    [EMPLOYMENT_VALUES.SELF_EMPLOYED, EMPLOYMENT_VALUES.UNEMPLOYED].some(status => status === employment_status);
```

Comment:
is there a specific reason for comparing constants with an accepted type string?
Types should be consisting to avoid conflicts I suppose

Reply:
Thanks. Will replace the string type